xlators/performance/md-cache/src/md-cache.c

```diff
@@ -1616,6 +1616,10 @@ mdc_mkdir_cbk(call_frame_t *frame, void *cookie, xlator_t *this, int32_t op_ret,
     }
 
     if (local->loc.parent) {
+        if (postparent && postparent->ia_nlink == 0) {
```

Comment:
Do we not need to invalidate the inode attributes?

Reply:
Thank you for your attention. The inode attribute was only invalidated at the mdcache layer, it should not be invalidated at the lower layer, that is, the afr layer changed the iatt attribute of the parent folder before returning it to the upper layer.